waspc/README.md

```diff
@@ -34,7 +34,32 @@ It can then also run that web app for you, deploy it (not yet but that is coming
 ### Setup
 We use the [`cabal`](https://cabal.readthedocs.io/) command line tool to build
 the project. The best way to install it is via
-[ghcup](https://www.haskell.org/ghcup/). The `ghcup tui` terminal user interface
+[ghcup](https://www.haskell.org/ghcup/) (if you're running an M1/M2 mac make sure to run it in rosetta).
+
+<details>
+<summary>M1/M2 rosetta setup</summary>
+
+1. Regular Shell: just run `arch -x86_64 zsh` in your terminal.
+2. IntelliJ: Preferences -> Tools -> Terminal -> Shell path -> `env /usr/bin/arch -x86_64 /bin/zsh --login`
```

Comment:
Are we ok with doing this for all of the shells that IntelliJ runs? Because this will then be also used when you are not dealing with Haskell, right? Same question for vscode below.

Reply:
This sets it per project (so only in this folder -> `.idea` is the config for it) and same for vscode (I suggested creating the `.vscode/settings.json` to make it local)